WordPress/Classes/ViewRelated/Comments/Controllers/CommentDetailViewController.swift

```diff
@@ -428,14 +426,16 @@ private extension CommentDetailViewController {
 
         // otherwise, if this is a comment to a post, show the post title instead.
         headerCell.configure(for: .post, subtitle: comment.titleForDisplay())
-        headerCell.contentView.heightAnchor.constraint(greaterThanOrEqualToConstant: 54).isActive = true
     }
 
     func configureContentCell(_ cell: CommentContentTableViewCell, comment: Comment) {
         let viewModel = CommentCellViewModel(comment: comment, notification: notification)
 
         cell.configure(viewModel: viewModel, helper: helper) { [weak self] _ in
-            self?.tableView.performBatchUpdates({})
+            guard let self else { return }
+            UIView.setAnimationsEnabled(false)
+            self.tableView.performBatchUpdates({})
+            UIView.setAnimationsEnabled(true)
```

Comment:
Nitpick: Does `UIView.performWithoutAnimation` work?

Reply:
I tested it with `UIView.setAnimationsEnabled`, so let's keep it.